### DISCUSSION TITLE
How to use the Term store properties within search, without the ID of the term store.

### DISCUSSION THREAD

**Discussion by Stievo9997** [microsoft-search/pnp-modern-search#2178] [Q&A] [answered]
When using the term store data within search and as filter, but all relevant properties start with the ID of the term store. Is there a easy way to get only the terms without the ID in it.

![image](https://user-images.githubusercontent.com/108119115/175488011-522b7dd1-c3da-4626-8289-b3e7c1c4d89b.png)
![image](https://user-images.githubusercontent.com/108119115/175488057-a9f8bd0f-80b6-41f4-bc70-00260f363171.png)
![image](https://user-images.githubusercontent.com/108119115/175488092-20d0fe70-18db-4296-924a-6da30f00eeed.png)

 Any help will be appreciated.

**Comment by saregbesola** [comment, marked as answer]
> When using the term store data within search and as filter, but all relevant properties start with the ID of the term store. Is there a easy way to get only the terms without the ID in it.
> 
> ![image](https://user-images.githubusercontent.com/108119115/175488011-522b7dd1-c3da-4626-8289-b3e7c1c4d89b.png) ![image](https://user-images.githubusercontent.com/108119115/175488057-a9f8bd0f-80b6-41f4-bc70-00260f363171.png) ![image](https://user-images.githubusercontent.com/108119115/175488092-20d0fe70-18db-4296-924a-6da30f00eeed.png)
> 
> Any help will be appreciated.

You can change that by going to Layout slots and change the slot field value from owstaxidmetadataaiitagsinfo to the managed property you mapped to in the search schema.

**Reply by Stievo9997** [reply]
Thanks! Didn't know that. Works as expected now.
